lib/patches/message_patch.rb

```diff
@@ -0,0 +1,24 @@
+require_dependency 'issue'
+
+module Patches
+  module MessagePatch
+    def self.included(base)
+      base.send(:include, InstanceMethods)
+      base.send(:before_create, :create_read_by_users)
+    end
+
+    module InstanceMethods
+      def create_read_by_users
+          # TODO dana: for all messages? no filter?
```

Comment:
Is this TODO still a todo?

Reply:
Is not a to do anymore. For the journals, we filter them by picking only those that are notes, but for messages I don't see why we should exclude some.